xml/dm_registers.xml

```diff
@@ -540,6 +540,46 @@
         <field name="data" bits="31:0" access="R/W" reset="0" />
     </register>
 
+    <register name="Advanced" short="advanced" address="0x32">
```

Comment:
Can we give this a name something more descriptive of what is in it than "advanced". At the very least, can it be "advancedcs" for "advanced control and status" which was more following our convention? There is no description for this register. What is envisioned for current and/or future contents of the "Advanced" register?

Reply:
Yes, but I'm not sure what a better name is. I think of this as a second DM control and status register. Except there is no first one, because it's separated into dmcontrol and dmstatus.

Should I split this into dmcontrol2 and dmstatus2? Call it dmcs2?

Reply:
dmcs2 makes sense to me

Reply:
I've renamed it.

Reply:
Do we usually write descriptions for the registers themselves (not just the fields)? Should there be one for this new register?

Reply:
I had added:
This register contains DM control and status bits that didn't easily
        fit in \Rdmcontrol and \Rdmstatus, and are only likely to be
        implemented in larger systems.